### PR TITLE
LPS-199847 | Poshi Automation | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
# LPS-198257 - Product QA | Test Automation Creation

Hi! :)
This pr is corresponding to this ticket: [LPS-198257](https://liferay.atlassian.net/browse/LPS-198257) → [LPS-199847](https://liferay.atlassian.net/browse/LPS-199847)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/pageaudit/PageAudit.path [hunk 1]
```diff
@@ -140,6 +140,11 @@
 	<td>//div[contains(@class,'page-audit')]//button[contains(@aria-label,'${cardName}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>PAGE_AUDIT_EXPERIENCE_SELECTOR</td>
+	<td>//*[contains(@aria-labelledby,'tab-performance')]//button[contains(@aria-label,'Experience Selector')]</td>
+	<td></td>
+</tr>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/pageaudit/PageAudit.path:147]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/pageaudit/PageAudit.testcase [hunk 2]
```diff
@@ -1297,6 +1297,59 @@ definition {
...
+				locator1 = "Fragment#CONTRIBUTED_FRAGMENT_TEXT",
+				value1 = "Go Somewhere");
+
+			AssertElementPresent(locator1 = "PageAudit#PAGE_AUDIT_BUTTON");
+
+			PageAudit.openPageAuditSidebar();
+		}
+
+		task ("Then it is possible to select the experience through the performance tab") {
+			Click(locator1 = "PageAudit#PAGE_AUDIT_EXPERIENCE_SELECTOR");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/pageaudit/PageAudit.testcase:1340]: `Button#ANY_WITH_ARIA_LABEL`

**Review by Tim-Cao — CHANGES_REQUESTED** [review]
Hi @rafaellajordao, would you mind updating according to comments? Thanks

**Comment by Tim-Cao** [comment]
@rafaellajordao see changes https://github.com/liferay-echo/liferay-portal/pull/14327

**Pull request closed without merging**